index.html

```diff
@@ -19,22 +19,41 @@
       var theme = 'dark';
       function init() {
             window.negotiator = new negotiator.Client({
-                type: 'active',
```

Comment:
Could you please revert all changes to index.html? You can keep your version for testing just don't commit it.

Reply:
yes